built/std_nodes/file_reader_spout.d.ts

```diff
@@ -1,36 +1,47 @@
 import * as intf from "../topology_interfaces";
 /** This spout reads input file in several supported formats and emits tuples. */
-export declare abstract class StringReaderSpout implements intf.Spout {
+export declare class FileReaderSpout2 implements intf.Spout {
```

Comment:
This is an alternative implementation that I would not like to loose, just in case.
I commented it out for now.